sunpy/net/vso/tests/test_vso.py

```diff
@@ -359,3 +359,49 @@ def test_iris_filename(client):
     search_results = client.search(a.Time("2018-01-02 15:31:55", "2018-01-02 15:31:55"), a.Instrument.iris)
     filename = client.mk_filename(pattern, search_results[0], None, url)
     assert filename.endswith("iris_l2_20180102_153155_3610108077_sji_1330_t000_fits.gz")
+    
+@pytest.mark.remote_data
+def test_table_noinfo_required(client):
+    res = client.search(a.Time('2017/12/17 00:00:00', '2017/12/17 06:00:00'), a.Instrument('aia'),
+                        a.Wavelength(171 * u.angstrom),
+                        response_format="table")
+    has_info = False
+    res_keys = res.keys()
+    if 'Info Required' in res_keys:
+       has_info = True
+    assert not has_info
+
+@pytest.mark.remote_data
+def test_table_has_info_required_swap(client):
+    res = client.search(a.Time('2020/02/15 00:00:00', '2020/02/15 20:00:00'), a.Instrument('swap'), a.Provider('ESA'), a.Source('PROBA2'),
+                        response_format="table")
+    has_info = False
+    res_keys = res.keys()
+    if 'Info Required' in res_keys:
+       has_info = True
+    assert has_info
+
+@pytest.mark.remote_data
+def test_table_length_test_swap(client):
```

Comment:
Does this test actually test anything in the sunpy code base? You haven't modified anything in search to make REST/TAP work?

I am a little nervous about tests which assert the number of results like this as that number can change without any modification to sunpy.

Reply:
The length tests in SunPy indirectly test that the VSO indeed returns the expected amount of data from specific TAP queries. I have similar tests in the backend Perl code testing the main CGI that the WSDL points to. We can of course take out the length tests if you prefer. Given that both the SWAP and LYRA tests are for dates back in 2020, I would think they are most likely done with any calibration and that the no. of files returned should be stable.